server-api/src/main/proto/sonarqube/ws-issues.proto

```diff
@@ -56,6 +56,11 @@ message Issue {
   optional string creationDate = 23;
   optional sonarqube.ws.commons.RuleType type = 27;
   optional string ruleDescriptionContextKey = 37;
+  // skipping unused fields 38-39
+  optional sonarqube.ws.commons.CleanCodeAttribute cleanCodeAttribute = 40;
+  optional sonarqube.ws.commons.CleanCodeAttributeCategory cleanCodeAttributeCategory = 41;
```

Comment:
Is this used at all? I don't see it used when handling API result. Same for the `TaintVulnerabilityLite` message

Reply:
Indeed, this field is not used by SLCORE, I believe we can safely skip it.